.github/workflows/llvmdev_win_builder.yml

```
@@ -8,35 +8,33 @@ jobs:
   windows:
     name: Windows
     runs-on: windows-2019
+    defaults:
+      run:
+        shell: bash -elx {0}
 
     steps:
-    - uses: actions/checkout@v4
-
-    - name: Install Miniconda
-      shell: pwsh
-      run: |
-        $wc = New-Object net.webclient
-        $wc.Downloadfile("https://repo.anaconda.com/miniconda/Miniconda3-latest-Windows-x86_64.exe", "Miniconda3-latest-Windows-x86_64.exe")
-        Start-Process "Miniconda3-latest-Windows-x86_64.exe" "/S /D=C:\Miniconda3" -Wait
-
-    - name: Setup Conda Environment
-      shell: bash
-      run: |
-        source /c/Miniconda3/Scripts/activate
-        conda create -n builder conda-build -y
-
-    - name: Build
-      shell: bash
-      run: |
-        source /c/Miniconda3/Scripts/activate
-        conda activate builder
-        RECIPE_NAME=./conda-recipes/llvmdev_manylinux
-        conda build $RECIPE_NAME
-        echo "OUTPUT=$(conda build --output $RECIPE_NAME)" >> $GITHUB_ENV
-
-
-    - name: Upload artifact
-      uses: actions/upload-artifact@v4
-      with:
-        name: llvmdev
-        path: ${{ env.OUTPUT }}
+      - name: Clone repository
+        uses: actions/checkout@v4
+
+      - name: Setup Miniconda
+        uses: conda-incubator/setup-miniconda@v3
+        with:
+          auto-update-conda: true
+          auto-activate-base: true
+          conda-remove-defaults: true
```

```suggestion
          activate-environment: ""
```

This change will not activate the default `test` environment. And also, we need `defaults` to get the compilers and `conda-build` and so on.